charts/oes/charts/spinnaker/templates/configmap/additional-profile-configmaps.yaml

```diff
@@ -12,7 +12,7 @@ Render profiles for each service by merging predefined defaults with values pass
 
 {{- /* Defaults: Disable S3 versioning on Front50 if Minio storage is used */}}
 {{- /* https://www.spinnaker.io/setup/install/storage/minio/#editing-your-storage-settings */}}
```

Comment:
If any of our subcharts are not consuming minio config, this can stay outside global